tests/integrationTests/inputs/compressibleFlow/turbulentFlow/turbulentChannelFlow.yaml

```diff
@@ -0,0 +1,208 @@
+#0 This case is inspired by the setup for Direct Numerical Simulation of turbulent channel flow up to Ret=590 (Moser et al. Physics of Fluids, 11(4):943–945, 1999).
+# The case is a statistically-developing internal flow through parallel smooth walls.
+# The domain comprises a 3D rectangular channel.
+# Longitudinal direction (Mean-flow direction)
+# Vertical direction (Wall-normal direction)
+# Spanwise direction (Statistically homogeneous direction).
+# Large eddy simulation (LES) is used to model sub-grid-scales and wall models are deployed near the wall regions.
+
+# Turbulent inflow is imposed at the intel. Turbulent velocity profile is approximated from 1/7 power law i.e., u = Umax (y/Re)^1/7 with synthetic turbulence generator.
```

Comment:
I think that if you wrap the equations with &&, they will show up as equations in the docs, you can preview this using either https://ablate.dev/content/development/Documentation.html#local-docs-preview or https://ablate.dev/content/development/Documentation.html#pull-request-preview